src/tools/rustc-perf-wrapper/src/main.rs

```diff
@@ -87,35 +105,65 @@ fn main() {
 
 fn run(args: Args) {
     let mut cmd = Command::new(args.ctx.collector);
+    let db_path = args.ctx.results_dir.join("results.db");
```

Comment:
It seems like we will need to create the `results_dir` if it does not exist, as I just found out. You can try it with `./x perf -- benchmark baseline`.

Reply:
I could confirm that situation. Thanks. Just fixed.